src/components/ActiveFilters.vue

```diff
@@ -92,11 +92,25 @@ export default Vue.extend({
           }
           // Range Filter
           if (filter.type === 'range-filter') {
-            activeValues.push({
-              key,
-              value: `${current[0]} to ${current[1]}`,
-              label: filter.label
-            })
+            if (current[0] == null && current[1] != null) {
+              activeValues.push({
+                key,
+                value: `${current[1]} and less`,
```

Comment:
'and less' i18n ? , or do we do this later in separate PR for all text ?

Reply:
There seems to be no i18n in the entire project.

Reply:
Added the point to:
https://wiki.gcc.rug.nl/ticket/7326